tests/Tests/ExchangerTest.php

```diff
@@ -320,6 +320,26 @@ public function it_supports_overrding_ttl_per_query()
 
     /**
      * @test
+     * @expectedException \Exchanger\Exception\CacheException
+     */
+    public function it_throws_an_exception_if_cache_key_is_too_long()
```

Comment:
Could you also please add a test to insure that the query `cache_prefix` overrides the `Exchanger` option ?